locales/ca/generate.json

```diff
@@ -0,0 +1,52 @@
+{
+   "meta": {
+      "description": "Genera llistes de música a Spotify basat en les cançons que estàn tocant en aquell tour, utilizant la API de Setlist.fm",
```

Comment:
```suggestion
      "description": "Genera llistes de música a Spotify basat en les cançons que estan tocant en aquell tour, utilizant la API de Setlist.fm",
```

Reply:
I also feel that this sentence feels a bit confusing. "que estan tocant" does not have a subject in the sentence. What about something like:

"Genera llistes amb les cançons que les teves bandes estan girant actualment"

Note that I am excluding de implementation details (e.g. Spotify, Setlistfm).